urbanaccess/gtfs/network.py

```diff
@@ -124,55 +141,89 @@ def create_transit_net(gtfsfeeds_dfs, day,
         gtfsfeeds_dfs.stop_times_int = _time_difference(
             stop_times_df=gtfsfeeds_dfs.stop_times_int)
 
+    # direction_id is conditional, though
+    if 'direction_id' not in gtfsfeeds_df.trips.columns:
+        columns.remove('direction_id')
+    
+    trips_subgdf = gtfsfeeds_df.trips[columns]
+    calendar_selected_trips_df = tripschedualselector(
+                                    input_trips_df=trips_subgdf,
+                                    input_calendar_df=gtfsfeeds_df.calendar,
+                                    day=day)
```

Comment:
You are missing one required kwarg here: `input_calendar_dates_df=gtfsfeeds_dfs.calendar_dates,`